packages/dashboard-frontend/src/services/helpers/types.ts

```diff
@@ -86,7 +86,7 @@ export enum LoaderTab {
 export enum WorkspaceDetailsTab {
   OVERVIEW = 'Overview',
   DEVFILE = 'Devfile',
-  LOGS = 'Logs',
+  DEVWRKSPACE = 'Devworkspace',
```

Comment:
let's call it `DevWorkspace`
![image](https://user-images.githubusercontent.com/1461122/206233522-1f5660a0-8172-4ca4-9776-d1dce18d16d5.png)

Reply:
Ok. I have fixed it.